Assets/MixedRealityToolkit/_Core/EventDatum/Input/MixedRealityPointerEventData.cs

```diff
@@ -31,10 +32,11 @@ public class MixedRealityPointerEventData : InputEventData
         /// </summary>
         /// <param name="pointer"></param>
         /// <param name="inputAction"></param>
+        /// <param name="inputSource"></param>
         /// <param name="count"></param>
-        public void Initialize(IMixedRealityPointer pointer, MixedRealityInputAction inputAction, int count = 0)
+        public void Initialize(IMixedRealityPointer pointer, MixedRealityInputAction inputAction, IMixedRealityInputSource inputSource = null, int count = 0)
         {
-            Initialize(pointer.InputSourceParent, inputAction);
+            BaseInitialize(pointer.InputSourceParent, inputAction);
```

Comment:
Looks like the check for `inputSource ?? pointer.InputSourceParent` was removed.

Reply:
Good catch, Thanks